tools/build_defs/repo/http.bzl

```diff
@@ -319,6 +321,12 @@ The archive will be unpacked into this directory, after applying `strip_prefix`
 (if any) to the file paths within the archive. For example, file
 `foo-1.2.3/src/foo.h` will be unpacked to `bar/src/foo.h` if `add_prefix = "bar"`
 and `strip_prefix = "foo-1.2.3"`.""",
+    ),
+    "files": attr.string_keyed_label_dict(
+        doc = """A map of relative paths (key) to a file label (value) that overlaid on the repo as
+a symlink. This is useful when you want to add WORKSPACE or BUILD.bazel files atop an existing
```

Comment:
`WORKSPACE` is deprecated, can you replace it with `REPO.bazel`? Maybe also update the doc for `remote_file_urls`?